packaging/greenboot/microshift-running-check.sh

```diff
@@ -19,6 +19,7 @@ trap 'script_exit' EXIT
 # The term handler to override the default behavior and have a uniform and
 # homogeneous exit code in all controlled situations.
 function return_failure() {
+    echo "Signal received, terminating."
```

Comment:
From the handler name, it looks like it's also invoked on "return 1" from functions. 
I'm not sure it's an appropriate message in all cases.